azure-quantum/azure/quantum/optimization/solvers.py

```diff
@@ -117,6 +137,29 @@ def set_one_param(self, name: str, value: Any):
             params = self.params["params"] if self.nested_params else self.params
             params[name] = str(value) if self.force_str_params else value
 
+    def check_set_schedule(self, schedule:RangeSchedule, schedule_name:str):
+        if not(schedule is None):
+            if not isinstance(schedule, RangeSchedule):
+                raise ValueError(f'{schedule_name} can only be from class "RangeSchedule"!')
+            schedule_param = {"type":schedule.type, "initial":schedule.initial, "final":schedule.final}
```

Comment:
Please add spaces after the colons in the dict, see PEP8 guidelines:
https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

```suggestion
            schedule_param = {"type": schedule.type, "initial": schedule.initial, "final": schedule.final}
```